src/MongoDB.Driver.Core/Core/WireProtocol/DeleteWireProtocol.cs

```diff
@@ -13,15 +13,16 @@
 * limitations under the License.
 */
 
+using System;
 using MongoDB.Bson;
 using MongoDB.Driver.Core.Connections;
 using MongoDB.Driver.Core.Misc;
-using MongoDB.Driver.Core.Operations;
 using MongoDB.Driver.Core.WireProtocol.Messages;
 using MongoDB.Driver.Core.WireProtocol.Messages.Encoders;
 
 namespace MongoDB.Driver.Core.WireProtocol
 {
+    [Obsolete("Use CommandWireProtocol instead.")]
```

Comment:
Consider deleting this class

Reply:
Same as for `DeleteOpcodeOperation` it is used in `Server.Delete` public method (https://github.com/mongodb/mongo-csharp-driver/blob/2ad99632b9af100060ce14288950114718b99e0c/src/MongoDB.Driver.Core/Core/Servers/Server.cs#L661) which is used in `DeleteOpcodeOperation` (https://github.com/mongodb/mongo-csharp-driver/blob/2ad99632b9af100060ce14288950114718b99e0c/src/MongoDB.Driver.Core/Core/Operations/DeleteOpcodeOperation.cs#L190).
Thus, if we do not delete `DeleteOpcodeOperation`, `DeleteWireProtocol` shouldn't be deleted as well.